<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="inputName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>71</width>
      <height>41</height>
     </size>
    </property>
    <property name="text">
     <string>Nume:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>761</width>
      <height>91</height>
     </rect>
    </property>
    <property name="searchPaths">
     <stringlist/>
    </property>
   </widget>
   <widget class="QLabel" name="readWords">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Citeste urmatoarele cuvinte:</string>
    </property>
   </widget>
   <widget class="QLabel" name="readWords_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cuvinte recunoscute:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>761</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="readWords_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Informatii</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>761</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="inregistrare">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Inregistrare</string>
    </property>
   </widget>
   <widget class="QPushButton" name="autentificare">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Autentificare</string>
    </property>
   </widget>
   <widget class="QPushButton" name="antrenare">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Antrenare</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="Menu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <addaction name="Menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
